### PR TITLE
Misc DB Fixes

### PR DESCRIPTION
**Pull request by NickTyrer**

### CODE REVIEW

**Comment by NickTyrer** [comment]
Build failing because gossip_menu_option is not allowing duplicate entrys.

**Comment by ratkosrb** [comment]
Why are you inserting a gossip menu combination that already exists?

**Comment by NickTyrer** [comment]
Its complainig about gossip_menu_option, which I think should be able to have duplicate menu_id

**Comment by ratkosrb** [comment]
There the key is menu_id + id (this is the index of the option in the list).

**Comment by NickTyrer** [comment]
Got it, however this condition type doesnt seem to work correctly:
https://github.com/vmangos/core/pull/1467/files#diff-85d8a19e2fa8ab06f7b4ca562ddb87c26d5abbdfe12cae7acf684f45fb7c5088R1474

File: sql/migrations/20220224185216_world.sql [hunk 1]
```diff
@@ -0,0 +1,2258 @@
...
+
+-- ----------------------------------------------------------------------
+
+-- Remove Creature Under Map
+DELETE FROM `creature` WHERE `guid` = 50190;
+
+-- ----------------------------------------------------------------------
+
+-- Set Correct Start Date For Southshore Assassins
+UPDATE `game_event` SET `start_time` = '2022-01-01 00:19:48' WHERE `entry`=152;
```
Comment (Kittnz) [inline sql/migrations/20220224185216_world.sql:258]: I really doubt it would be at 00:19:48, it's prolly a between time value

Comment (NickTyrer) [reply sql/migrations/20220224185216_world.sql:258]: I thought this was just an arbitrary value that enables the event?

File: sql/migrations/20220224185216_world.sql [hunk 2]
```diff
@@ -1360,8 +1360,8 @@ DELETE FROM `gameobject` WHERE `guid` = 73246;
 
 -- ----------------------------------------------------------------------
 
--- Correct Serpentbloom Flags
-UPDATE `gameobject_template` SET `flags` = 0,  `size` = 0.1 WHERE `entry` = 19535;
+-- Correct Serpentbloom Flags And Make Clickable
+UPDATE `gameobject_template` SET `flags` = 0,  `displayId` = 0 WHERE `entry` = 19535;
```
Comment (ratkosrb) [inline sql/migrations/20220224185216_world.sql]: displayid = 0 ??
wtf

File: sql/migrations/20220224185216_world.sql [hunk 3]
```diff
@@ -5933,15 +5933,15 @@ DELETE FROM `quest_end_scripts` WHERE `id`=5803;
 INSERT INTO `quest_end_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
 (5803, 0, 0, 4, 147, 7, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Araj\'s Scarab - Alchemist Arbington: npc\\_flag questgiver+gossip+vendor removed'),
 (5803, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7278, 0, 0, 0, 0, 0, 0, 0, 0, 'Araj\'s Scarab - Alchemist Arbington: Say text 1'),
-(5803, 3, 0, 15, 9744, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Araj\'s Scarab - Alchemist Arbington: Cast spell \"Psychometry\"'),
+(5803, 3, 0, 15, 9744, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Araj\'s Scarab - Alchemist Arbington: Cast spell'),
 (5803, 10, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 7279, 0, 0, 0, 0, 0, 0, 0, 0, 'Araj\'s Scarab - Alchemist Arbington: Say text 2'),
 (5803, 10, 0, 4, 147, 7, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Araj\'s Scarab - Alchemist Arbington: npc\\_flag questgiver+gossip+vendor added');
```
Comment (ratkosrb) [inline sql/migrations/20220224185216_world.sql]: whats up with the \\ in npc_flag here?

**Pull request merged**